src/components/Gradebook/index.jsx

```diff
@@ -433,13 +443,40 @@ export default class Gradebook extends React.Component {
                   open={this.state.modalOpen}
                   title="Edit Grades"
                   closeText="Cancel"
-                  body={(
+                  body={( // UI for this component
```

Comment:
Please remove this comment. If you don't that it is clear that the jsx there is for the component, please turn it into a React functional component that way it can be referenced by a name: https://reactjs.org/docs/components-and-props.html